packages/client/src/runtime/core/extensions/$extends.ts

```diff
@@ -61,8 +61,8 @@ type QueryOptionsCbArgs = {
 type ModelQueryOptionsCbArgs = {
   model: string
   operation: string
-  args: JsArgs
-  query: (args: JsArgs) => Promise<unknown>
+  args: unknown
```

Comment:
Why this change? `JsArgs` properly types all possible user input values.

Reply:
Yes, there is no reason to change this, just me testing something.